core/controllers/community_dashboard.py

```diff
@@ -76,3 +77,34 @@ def get(self, opportunity_type):
         }
 
         self.render_json(self.values)
+
+
+class TranslatableTextHandler(base.BaseHandler):
+    """Provides lessons content which can be translated in a given language."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.open_access
+    def get(self):
+        """Handles GET requests."""
+        language_code = self.request.get('language_code')
+        exp_id = self.request.get('exp_id')
+
+        if not utils.is_supported_audio_language_code(language_code):
+            raise self.InvalidInputException('Invalid language_code: %s' % (
+                language_code))
+
+        if not opportunity_services.is_exploration_available_for_contribution(
+                exp_id):
+            raise self.InvalidInputException('Invalid exp_id: %s' % exp_id)
+
+        exp = exp_fetchers.get_exploration_by_id(exp_id)
+        state_names_to_content_id_mapping = exp.get_translatable_text(
+            language_code)
+
+        self.values = {
+            'state_wise_contents': state_names_to_content_id_mapping,
```

Comment:
I thought we talked about changing the key name ("X-wise" is quite colloquial)

Reply:
Done!